index.html

```diff
@@ -7,65 +7,76 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
 <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
 <link href="https://fonts.googleapis.com/css2?family=Akshar:wght@500&family=Hind:wght@300&family=Lobster&family=Reenie+Beanie&family=Source+Sans+Pro:wght@200&display=swap" rel="stylesheet">
-    <link rel="stylesheet" href="index.css">
-    <script src="index.js"></script>
+<link rel="stylesheet" href="index.css">
+<script src="index.js"></script>
 <title>Personal Portfolio</title>
 </head>
 <body>
-    <div class="navigation">

```

Comment:
- [ ] [OPTIONAL] Always includes a `why`, an explanation for each requested change.
Here you can write:
Why don't you use a `nav` tag instead of a div, that would improve the HTML structure and make the page more accessible